src/local.lambda.ts

```diff
@@ -34,14 +34,15 @@ export class LocalLambda {
           response.end();
           return; // for complex requests(POST etc)' CORS header
         }
-
+        let body = Buffer.concat(data);
         const req: RequestEvent = {
           path: parsedUrl.pathname!,
           httpMethod: request.method as HTTPMethod,
           method: request.method,
           headers: request.headers,
           queryStringParameters: parsedUrl.query as Record<string, string>,
-          body: data,
+          body: body.toString('base64'),
+          isBase64Encoded: true,
```

Comment:
I think you should base the criteria of this being base64encoded based on content type or whether or not a flag is explicitly passed. 

To mimic the intent of passing binary data you could set it like how it’s described here in the docs:

`HTTP API Lambda integration can automatically infer the need for encoding based on the content-type header passed with the request. Passing an isBase64Encoded boolean value with the data simplifies the encoding and decoding of binary data.`

Reply:
Hi @blakejoy , I updated the PR accordingly. Please take a look and let me know. Thanks for quick feed back. 🙏